setup.py

```diff
@@ -210,7 +210,7 @@ def build_extension(self, ext):
         if build_gromacs:
             # TODO! We need to distinguish dev branch builds from master branch builds or always build with the
             # master branch of the dependency. For one thing, as is, this line needs to be toggled for every release.
-            gromacs_url = "https://github.com/kassonlab/gromacs-gmxapi/archive/dev_5.zip"
+            gromacs_url = "https://github.com/kassonlab/gromacs-gmxapi/archive/devel.zip"
```

Comment:
Is there a compelling reason to use a static zip file for gromacs_url rather than a git branch (or tag) checkout?

Reply:
It is much faster than cloning the repo and uses standard Python modules. More recently, I've found the fastest thing is `git clone -depth 1 ...`, but when we move off of readthedocs.org to a push from Travis-CI, we will not need to work so hard to shave seconds off the build time.